### PR TITLE
Added template loginsys

### PR DESCRIPTION
**Pull request by AlexanderDKazakov**
Changes to be committed:
	modified:   .gitignore
	new file:   apps/loginsys/templates/base.html
	deleted:    apps/loginsys/templates/index.html
	modified:   apps/loginsys/views.py

### CODE REVIEW

**Comment by shurik111333** [comment]
Why you rename `index.html` to `base.html`?

**Comment by AlexanderDKazakov** [comment]
I plan to extend base.html using {% include %} parts

File: apps/loginsys/templates/base.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,600' rel='stylesheet' type='text/css'>
+    <link href="{% static "css/toolkit.css"%}" rel="stylesheet">
+    <link href="{% static "css/application.css"%}" rel="stylesheet">
+
+    <style>
+      body {
+        width: 1px;
+        min-width: 100%;
+        *width: 100%;
+      }
```
Comment (shurik111333) [inline apps/loginsys/templates/base.html]: wat?

File: apps/loginsys/templates/base.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+</div>
+
+    <script src="{% static "js/jquery.min.js" %}"></script>
+    <script src="{% static "js/popper.min.js" %}"></script>
+    <script src="{% static "js/chart.js" %}"></script>
+    <script src="{% static "js/toolkit.js" %}"></script>
+    <script src="{% static "js/application.js" %}"></script>
+
+    <script>
+      $(function(){while(window.BS&&window.BS.loader&&window.BS.loader.length){(window.BS.loader.pop())()}})
```
Comment (shurik111333) [inline apps/loginsys/templates/base.html]: What is window.BS? Does this work in any browser and OS?

File: apps/loginsys/views.py [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 
 def welcome(request):
-    return render(request=request, template_name="index.html")
+    return render(request=request, template_name="base.html")
 
 #TODO: There are several function, which we need to do.
 #
```
Comment (shurik111333) [inline apps/loginsys/views.py]: This is uninformative TODO. Remove it please

**Pull request closed without merging**